firedantic/tests/tests_async/test_model.py

```diff
@@ -1,6 +1,10 @@
+from operator import attrgetter
 from uuid import uuid4
 
 import pytest
+from google.cloud.firestore import (  # just for the ASCENDING / DESCENDING constants
+    Query,
+)
```

Comment:
Why is this comment and poor format needed?

```suggestion
from google.cloud.firestore import Query
```